src/x/pool/options.go

```diff
@@ -36,10 +42,21 @@ type objectPoolOptions struct {
 	onPoolAccessErrorFn OnPoolAccessErrorFn
 }
 
+func getDefaultSize() int {
+	v, ok := os.LookupEnv("AIVEN_POOL_DEFAULT")
+	if ok {
```

Comment:
I've grown to like the "comma ok" idiom _(a.k.a someones been reading "effective go")_... So maybe...
```golang
if v, ok := os.LookupEnv("AIVEN_POOL_DEFAULT"); ok {
...
}`

Reply:
Personally not super fond of the format as it is easy to have errors on too long lines, but don't care much either way either so changing 😏